server/middleware/auth.js

```diff
@@ -9,6 +20,79 @@ module.exports.verify = (req, res, next) => {
   res.redirect('/login');
 };
 
+module.exports.verifyElse401 = (req, res, next) => {
+  if (req.isAuthenticated()) {
+    return next();
+  }
+  res.sendStatus(401);
+};
+
+module.exports.verifyBoardMemberElse401 = (req, res, next) => {
+  var boardid;
+  if (!req.params && !req.body) {
+    res.status(400).send('board id couldnt be found in request from client');
+  }
+  if (!req.params.id) {
+    boardid = parseInt(req.body.id);
+  } else {
+    boardid = parseInt(req.params.id);
+  }
+  var userId = req.user.id;
+
+  // return models.User.where({ id: userId }).fetch({withRelated:['memberOfBoards']})
+  // .then(function(user) {
+  //   return user.related('memberOfBoards');
+  // })
+
+  //see whether the boardid shows up under any of the users boards
```

Comment:
everything below up to .filter can be handled by getBoardsByUser

Reply:
Changed to getBoardsByUser